lib/domain-shared/app_constants.dart

```diff
@@ -19,3 +19,6 @@ const String chainId = '4667b205c6838ef70ff7988f6e8257e8be0e1284a2f59699054a018f
 
 /// Unplanted minimum balance requirement
 const double minPlanted = 5.0;
+
+/// Region ID extension
+const String regionIdExtension = ".rgn";
```

Comment:
regionIdPostfix or regionIdSuffx make more sense

Reply:
good Idea, renamed to regionIdSuffix